.github/workflows/snyk.yml

```
@@ -0,0 +1,29 @@
+name: Security Scanning
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest
```

actionlint (1.7.7)

key "runs-on" is duplicated in "security" job. previously defined at line:10,col:5

(syntax-check)